### ISSUE TITLE
Red Green Uptime Colors -> Red Blue (or some other scheme)

### ISSUE THREAD

**Issue by bhuvy2** [illinois-cs241/broadway-on-demand#61]
Just for accessibility, red green colorblindness is pretty common, and it should be a quick fix unless we want to delve into which shades of colors to use.

**Comment by shreyas208** [comment]
Fix deployed.
